src/benchmarkRunner/BenchmarkRunner.ts

```diff
@@ -90,14 +125,20 @@ class BenchmarkRunner extends BenchmarkCollection {
 				}
 			}
 
-			operationsPerSecond.sort((a, b) => a.ops - b.ops);
-			const base = operationsPerSecond[0];
-			for (let i = 0; i < operationsPerSecond.length; i++) {
-				const ops = operationsPerSecond[i];
-				if (i === 0) {
-					console.log(`${ops.name} 100%`);
-				} else {
-					console.log(`${ops.name} ${(ops.ops / base.ops) * 100}%`);
+			if (isLogExcellable) {
+				console.log('');
+			}
+
+			if (!isLogExcellable) {
+				operationsPerSecond.sort((a, b) => a.ops - b.ops);
```

Comment:
It might be better to make this one consistent with the CSV version. So have both just compare it to the first instead of this one marking the lowest one as 100%.